be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -163,7 +163,9 @@ Status ExchangeSinkOperator::Channel::send_one_chunk(const vectorized::Chunk* ch
     // last packet
     if (_current_request_bytes > _parent->_request_bytes_threshold || eos) {
         request.set_eos(eos);
-        TransmitChunkInfo info = {this->_channel_id, std::move(request), _brpc_stub};
+        auto attachment = std::make_shared<butil::IOBuf>();
```

Comment:
attachment, suggest to use a bare object, no need to use shared_ptr, since IOBuf already has refcount inside.